pkg/backend/path_creds.go

```diff
@@ -81,14 +81,16 @@ func (b *backend) credsUpdateOperation(ctx context.Context, req *logical.Request
 		}
 
 		tok, err = cb.Build().Exchange(ctx, code.(string))
-		if _, ok := err.(*oauth2.RetrieveError); ok {
+		if rErr, ok := err.(*oauth2.RetrieveError); ok {
+			b.logger.Error("invalid code", rErr)
```

Comment:
I think these are actually key/value pairs (not sure I've used this correctly elsewhere either...), so it should be something like `Error("invalid code", "error", rErr)`.

Reply:
Right, I'm sorry I forgot that.  Fixed.